tests/__init__.py

```diff
@@ -23,3 +35,34 @@ def execute_plugin_tasks(plugin: Task, verbosity: int = 0):
         catched = t.execute(stream)
         if catched:
             raise Exception("Task error for '{}'\n{}".format(t.name, catched.get_msg()))
+
+
+class regex:  # noqa
```

Comment:
This should be documented and named better. (Also, I’m not 100% sure if we need this.)

Reply:
removed